pynbody/halo/rockstar.py

```diff
@@ -243,7 +243,7 @@ def __init__(self, sim, dummy=False, filename=None, format_revision=None, **kwar
             raise IOError("Halo catalogue not found -- check the file name of catalogue data or try specifying a catalogue using the filename keyword")
 
         with f:
-            self._head = np.fromstring(f.read(self.head_type.itemsize),
+            self._head = np.frombuffer(f.read(self.head_type.itemsize),
```

Comment:
The numpy doc recommends using `np.frombuffer` when reading-in binary data (as is the case here). See https://numpy.org/doc/stable/reference/generated/numpy.fromstring.html.